pelican/generators.py

```diff
@@ -400,7 +402,8 @@ def generate_feeds(self, writer):
                     writer.write_feed(
                         arts,
                         self.context,
-                        self.settings['TAG_FEED_ATOM'].format(tag.slug),
+                        self.settings['TAG_FEED_ATOM'].replace('%s', '{slug}')
+                                                      .format(tag.slug),
```

Comment:
Why not `.format(slug=tag.slug)` here (and the two cases below)?

Reply:
Good catch. It's been fixed!